plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -33,7 +33,7 @@ const (
 	spanIndexName         = "jaeger-span-2019-01-01"
 	serviceIndexName      = "jaeger-service-2019-01-01"
 	indexCleanerImage     = "jaegertracing/jaeger-es-index-cleaner:latest"
-	rolloverImage         = "jaegertracing/jaeger-es-rollover:latest"
+	rolloverImage         = "pavolloffay/jaeger-es-rollover:PR5704"
```

Comment:
The `latest` image does not have the index templates for `ES 6` so the tests would fail.

I will revert this back to `latest` once the PR is merged and the latest image is updated.